src/components/FamilyManifestModal/FamilyManifestModal.js

```diff
@@ -257,10 +260,11 @@ export default compose(
                       const FooterWithParticipantsAndFamilyMembers = createFooterComponent(
                         uniqueParticipantsAndFamilyMemberIds,
                       );
-                      return loading ? (
+                      return loading || !fileTypeStats.length ? (
```

Comment:
this seems a little odd too, is a lack of length here actually a loading state? what if there are legitimately no file type stats?